.gitlab-ci.yml

```diff
@@ -338,14 +338,14 @@ TCID-DIR-DMAAS-SCHD-LOCAL-HP-MINIO-BUCKET:
     GROUP: dmaasHostpathMinio
     THREADS: 5
 
-# TCID-DIR-DMAAS-SCHD-LOCAL-DEV-MINIO-BUCKET:
-#   extends: .gui_dmaas_test_template
-#   script:
-#     - chmod 755 ./stages/14-director-gui-check/gui-dmaas-device-minio
-#     - ./stages/14-director-gui-check/gui-dmaas-device-minio
-#   variables:
-#     GROUP: dmaasDeviceMinio
-#     THREADS: 5
+TCID-DIR-DMAAS-SCHD-LOCAL-DEV-MINIO-BUCKET:
+  extends: .gui_dmaas_test_template
+  script:
+    - chmod 755 ./stages/14-director-gui-check/gui-dmaas-device-minio
+    - ./stages/14-director-gui-check/gui-dmaas-device-minio
+  variables:
+    GROUP: dmaasDeviceMinio
+    THREADS: 5
```

Comment:
Please explain these variables

Reply:
these are the variables used used by Selenium .
Group variable is marker which is tied to selenium test case . When Job will run, it will look for test case which is marked as `dmaasDeviceMinio`.